tutorials/math/beziers_and_curves.rst

```diff
@@ -34,6 +34,14 @@ change the value of ``t`` from 0 to 1.
         var q0 = p0.linear_interpolate(p1, t)
         var q1 = p1.linear_interpolate(p2, t)
 
+ .. code-tab:: csharp
+    
+    public Vector2 Quadratic_Bezier(Vector2 p0, Vector2 p1, Vector2 p2, float t)
+    {
+        var q0 = p0.LinearInterpolate(p1, t);
+        var q1 = p1.LinearInterpolate(p2, t);
```

Comment:
`linear_interpolate` and `LinearInterpolate` are `lerp` and `Lerp` in the master branch of Godot (what will become Godot 4.0). The GDScript example also needs updating, which would be appreciated :)

```suggestion
        Vector2 q0 = p0.Lerp(p1, t);
        Vector2 q1 = p1.Lerp(p2, t);
```